e2e/support/helpers/e2e-dashboard-helpers.js

```diff
@@ -248,3 +252,23 @@ export const getNextUnsavedDashboardCardId = (() => {
 export const dashboardHeader = () => {
   return cy.findByTestId("dashboard-header");
 };
+
+export const dashboardGrid = () => {
+  return cy.findByTestId("dashboard-grid");
+};
+
+export function createDashboardWithTabs({
```

Comment:
This function is based off the one in the original PR, but change the way we call the API because I suspect the tabs API back then was different. So the util isn't compatible.